doajtest/unit/test_reporting.py

```diff
@@ -14,14 +14,18 @@
     ["User", "2015-01", "2015-02", "2015-03", "2015-04", "2015-05", "2015-06"],
     ["u1", 0, 1, 2, 3, 4, 5],
     ["u2", 6, 7, 8, 9, 10, 0],
-    ["u3", 11, 12, 13, 14, 15, 16]
+    ["u3", 11, 12, 13, 14, 15, 16],
+    ["u4", 2, 5, 0, 8, 13, 21],
+    ["u5", 2, 5, 0, 8, 0, 4],
```

Comment:
Simply adding these 2 lines to the test data caused the test to fail as I expected, so well written test :).